readme.html

```diff
@@ -1,29 +1,468 @@
+                </ul>
+                <li><b>Action Parameter: Protocol</b></li>
+                <ul>
+                    <li>This parameter allows the user to select TCP or UDP as protocol for traffic. The default value
+                        is TCP.</li>
+                </ul>
+                <li><b>Action Parameter: Policy Decisions</b></li>
+                <ul>
+                    <li>This parameter is used to filter out the traffic based on policy decisions. An error message
+                        will be shown if the policy decision is invalid.</li>
+                </ul>
+                <li><b>Examples:</b>
+                    <ul>
+                        <li>Retrieve traffic for between 01 July, 2022 to 10 July, 2022 on 22 TCP and having policy
+                            decision potentially_blocked.
+                            <ul>
+                                <li>Start Time = "2022-07-01T17:45:08" </li>
+                                <li>End Time = "2022-07-10T17:45:08"</li>
+                                <li>Port = 22</li>
+                                <li>Protocol = "TCP"</li>
+                                <li>Policy Decisions = "potentially_blocked</li>
+                            </ul>
+                        </li>
+                    </ul>
+                </li>
+            </ul>
+        </li>
+        <li>
+            <h3>Create Virtual Service</h3>
+            Creates a virtual service.
+            <ul>
+                <li><b>Action Parameter: Port</b></li>
+                <ul>
+                    <li>This parameter will accept the port value for a virtual service. It expects a numeric value as
+                        an input. An error message will be shown if the port value is invalid.</li>
+                </ul>
+                <li><b>Action Parameter: Protocol</b></li>
+                <ul>
+                    <li>This parameter allows the user to select TCP or UDP from dropdown as protocol for virtual
+                        service</li>
+                </ul>
+                <li><b>Action Parameter: Name</b></li>
+                <ul>
+                    <li>This parameter will accept the name for a virtual service.</li>
+                </ul>
+                <li><b>Examples:</b>
+                    <ul>
+                        <li>Create new virtual service with name "test-vs" on port 22 and protocol "TCP".
+                            <ul>
+                                <li>Port = 22</li>
+                                <li>Protocol = "TCP"</li>
+                                <li>Name = "test-vs"</li>
+                            </ul>
+                        </li>
+                    </ul>
+                </li>
+            </ul>
+        </li>
+        <li>
+            <h3>Create Service Binding</h3>
+            <p>
+                Binds one or more workloads to active service. The virtual service must be created in the draft and then
```

Comment:
I think saying that it needs to be created in draft here is just going to be confusing - maybe just `The virtual service must be provisioned into active state to bind a workload to it.`

Reply:
Done